### PR TITLE
[Migrate to SQLAlchemy ORM - PART 4] Modify controller for reviewing applications to use SQLAlchemy models

### PR DESCRIPTION
**Pull request by thomasmunduchira**

### CODE REVIEW

File: src/controller/review.py [hunk 1]
```diff
@@ -1,37 +1,66 @@
...
-    return Response(str(data))
-
-@review.route('/review/skip')
-def skipApplicant():
-    model = ApplicationsModel.get_applications_model()
+@review.route('/review', methods=['GET'])
+@login_required
+def get_application():
+    """Get application for user"""
+    application = Application.get_application(current_user.get_id())
```
Comment (klanmiko) [inline src/controller/review.py]: is current_user filled by @login_required?

Comment (thomasmunduchira) [reply src/controller/review.py]: current_user is storing a generated user ID, there aren't any ties to the User object in the db.

File: src/controller/review.py [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
+        abort(400, 'score not an integer')
+
+    if score < 1 or score > 5:
+        abort(400, 'score not between 1 and 5')
+
+    application = Application.update_score(current_user.get_id(), score)
+    if application is None:
+        abort(400, 'User is not currently assigned an application')
+
+    return jsonify(Serializer.serialize_value(application))
```
[the remark this answers is not in the document]
Comment (thomasmunduchira) [reply src/controller/review.py]: Done

File: src/controller/review.py (hunk 1 above)
Comment (thomasmunduchira) [reply src/controller/review.py]: I take that back. In `auth.py`, there is a load_user function that loads in the current user object.

Comment (klanmiko) [reply src/controller/review.py]: flask-login. The answer is flask-login

**Pull request merged**